hack/generate.sh

```diff
@@ -53,9 +63,13 @@ parse_flags() {
         MODE="$1"
         fi
         ;;
-      --which)
+      --codegengroups)
+        shift
+        CODEGEN_GROUPS="${1:-$CODEGEN_GROUPS}"
+        ;;
+      --manifestsfolders)
```

Comment:
```suggestion
      --manifests-dirs)
```

Folder is more Windows/GUI term, dir/directory sounds more linux. I would vote to change to manifest dirs everywhere.